mw-webapp/src/component/errorBoundary/ErrorBoundary.tsx

```diff
@@ -0,0 +1,45 @@
+import React, {Component} from "react";
+import {Page404} from "src/pages/page404/Page404";
+
+/**
+ *   They are React components that catch JavaScript errors anywhere in their child component tree,
+ * log those errors, and display a fallback UI instead of the component tree that crashed.
+
+     It’s important to note that error boundaries catch errors during rendering, in lifecycle methods,
+   and constructors of the whole tree below them. However, error boundaries do not catch errors for:
+
+   Event handlers (for that, you need to use regular try/catch)
+   Asynchronous code (e.g., setTimeout or requestAnimationFrame callbacks)
+   Server-side rendering
+   Errors thrown in the error boundary itself (rather than its children).
```

Comment:
could you provide an example of error which will be catched by this component?

Reply:
Added. Will this example work?